### ISSUE TITLE
Callback when gif is loaded

### ISSUE THREAD

**Issue by youfacepalm** [koral--/android-gif-drawable#246]
Hey guys. Thanks for the great library. It would be great if there is a way to get a callback once the gif is loaded into a GifImageView.

**Comment by koral--** [comment]
What exactly do you mean by loaded? First frame is drawn?

**Comment by youfacepalm** [comment]
Yes. For example if I set a GifImageView to cover the entire screen with 'scaleType:centerCrop' it takes about 600ms for the first frame to appear (depending on the device).  I thought having a listener of some kind may help in this scenario. 

In my project I use GifImageView in splash screen. I needed to animate in my logo once the gif starte playing. Currently I am hard coding the animation delay to 600ms which seems inconsistent across various devices.

**Comment by RNOVOSELOV** [comment]
Hey, it would be nice to get a callback when finished playing (when the last frame is drawn)
thanks

**Comment by koral--** [comment]
@RNOVOSELOV callback when animation finished playing is already available. See sample project: https://github.com/koral--/android-gif-drawable-sample/blob/9840f2eab0d568f39c3c92d12121d52ce3391568/sample/src/main/java/pl/droidsonroids/gif/sample/AnimationControlFragment.java

**Comment by RNOVOSELOV** [comment]
Oy, thanks!!!
